### ISSUE TITLE
[$20][MSFT-194] [Registration]: In Keyboard navigation focus is lost after "Create password" field and interactive elements are not reachable from that point

### ISSUE THREAD

**Issue by veshu** [topcoder-platform/community-app#2860]
**Steps to Reproduce**
1. Launch https://accounts.topcoder.com/member/registration 
2. Navigate through TAB keys

**Expected Result**
While navigating through keyboard, the visual indicator should not be lost at any point of time. And, all interactive elements should be reachable through keyboard.

**Actual Result**
Note that visual focus is lost after "Create password" field
Hide button and other links are NOT reachable through "Keyboard"

**Device**: Desktop/Labtop
**Operating System**: macOS
**Browser**: Chrome

**WCAG Category**: 1. Perceivable
**WCAG Level**: A
**WCAG Success Criterion**: 2.4.3 Focus Order
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/pd-topcoder_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=1a9ec9e4%2D985a%2D449c%2D9e7a%2Dbb83cbe69ccb&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fpd%2Dtopcoder%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2FMSFT%2DTeams%2DQA%2FJune%202019%2F29%20Jun%2FTest%20Results%2FBug%20Videos%20%26%20Log%20Files%2FArun%5FA11Y%2FKeyboard%20navigation%20focus%20is%20lost

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096259 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096259 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30096259 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097681 has been created for this ticket.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by cwdcwd** [comment]
Contest https://www.topcoder.com/challenges/30097681 has been updated - it has been assigned to Timor.<br/><br/>```This is an automated message for lazybaer via Topcoder X```

**Comment by romitgit** [comment]
PR for this issue : https://github.com/appirio-tech/accounts-app/pull/265

**Comment by Oanh-and-only-Oanh** [comment]
@romitgit, please fix conflicts.

**Comment by romitgit** [comment]
@Oanh-and-only-Oanh Fixed conflicts and updated the request.

**Comment by romitgit** [comment]
@Oanh-and-only-Oanh https://github.com/appirio-tech/accounts-app/pull/265

**Comment by mtwomey** [comment]
This appears to have been fixed along the way - it's looks good to me in production?

**Comment by nkumar-topcoder** [comment]
this is already in prod. @r0hit-gupta @drasticdpk @lakshmiathreya can you please check and close this.

**Comment by crazyk07** [comment]
Payment task has been updated: https://software.topcoder.com/review/actions/ViewProjectDetails?pid=30097681<br/><br/>```This is an automated message for crazyk via Topcoder X```
